client/app/components/calendar/layout/SideCalendar.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import moment from 'moment';
+import Calendar from 'rc-calendar';
+import bindAll from 'lodash/bindAll';
+import 'rc-calendar/assets/index.css';
+import './side_calendar.scss';
+
+export default class SideCalendar extends React.Component {
+  static contextTypes = {
+    router: React.PropTypes.object
+  };
+
+  constructor(...args) {
+    super(...args);
+    this._setMondayAsFirstWeekDay();
+    this.state = { value: moment() };
```

Comment:
You should not store date here - there should be once source of truth about the date being currently displayed in the calendar - the date from the router query.
Inject it into `SideCalendar` as a prop